### PR TITLE
[trace repositories] harmonize configs, little improvements and clean…

### PR DESCRIPTION
**Pull request by marcdumais-work**
…-ups

We want to have a consistent configuration, for the org's repositories that we maintain. This means e.g. using the same "branch protection" rules across them. We've also disabled "merge commits", since we dislike their effect on the git history, and they were accidently used once in a while, despite us trying not-to.

Also did some little clean-ups and improvements, e.g. refreshed some of the repos "topics" and descriptions.

### CODE REVIEW

**Comment by netomi** [comment]
LGTM, if you add a project lead as reviewer and get an approval you can auto-merge yourself, instructions will be added.

I had to update the current config as some secrets were missing (they have been added manually it seems) and rebased the PR, but all looks good now.

Edit: I have to correct myself, as you delete some resources as well you wont be able to automerge, but once approval is there I will merge the PR.

File: otterdog/eclipse-cdt-cloud.jsonnet
Comment (netomi) [inline otterdog/eclipse-cdt-cloud.jsonnet]: requiring strict status checks but not status checks does not really make sense, as the strict checks are only taken into account when the status checks are enabled in the first place, you could also remove the strict status checks to avoid confusion.

Comment (marcdumais-work) [reply otterdog/eclipse-cdt-cloud.jsonnet]: Thanks - will fix this

Comment (marcdumais-work) [reply otterdog/eclipse-cdt-cloud.jsonnet]: done

**Review by netomi — APPROVED** [review]

**Comment by marcdumais-work** [comment]
(just pushed a minor update, updating only the commit message)

**Comment by marcdumais-work** [comment]
Thanks the the help and quick review @netomi!

**Comment by netomi** [comment]
np, actually, I did some unnecessary rebases that resulting in a lot of comments. The logic of the automated handling is that everytime the PR gets updated, the configuration change is validated and the live settings are checked if they are in sync. This can be further improved to reduce the noise.

**Comment by netomi** [comment]
btw. there is an updated dashboard available at https://otterdog.eclipse.org/projects/ecd.cdt-cloud where you can see the current config of your project in a more readable way.

**Comment by marcdumais-work** [comment]
> np, actually, I did some unnecessary rebases that resulting in a lot of comments. The logic of the automated handling is that everytime the PR gets updated, the configuration change is validated and the live settings are checked if they are in sync. This can be further improved to reduce the noise.

np - I think I triggered some of these myself :)

**Review by bhufmann — APPROVED** [review]
Looks good to me. Thanks!

**Pull request merged**
